tensorflow_addons/losses/sparsemax_loss.py

```diff
@@ -114,5 +114,7 @@ def __init__(self,
         if from_logits is not True:
             raise ValueError('from_logits must be True')
 
-        super(SparsemaxLoss, self).__init__(
-            sparsemax_loss_from_logits, name=name, reduction=reduction)
+        super(SparsemaxLoss, self).__init__(name=name, reduction=reduction)
+
+    def call(self, y_true, y_pred):
+        return sparsemax_loss_from_logits(y_true, y_pred)
```

Comment:
Should this also have it's own `get_config` method? The `from_logits` parameter doesn't seem like the best way to handle that situation, but for consistency if we have parameters we should serialize them.

Reply:
Sure, I'm fine with this one. Also cc @AndreasMadsen.

Reply:
`from_logits` Is a standardized parameters, that is often False by default. However for sparsemax there is no way of handling that situation. Thus it will just throw.

In isolation it looks pretty wired to have the parameter, but it appears in Softmax and other classification losses which is why it is there.